sql/item_mods.sql

```diff
@@ -29300,6 +29300,39 @@ INSERT INTO `item_mods` VALUES (25582,31,75);     -- Magic Evasion+75
 INSERT INTO `item_mods` VALUES (25582,68,63);     -- Evasion+63
 INSERT INTO `item_mods` VALUES (25582,165,4);     -- Crit Hit Rate+4
 INSERT INTO `item_mods` VALUES (25582,384,81);    -- Haste+81
+INSERT INTO `item_mods` VALUES (25583,1,91);      -- Mallquis Chapeau DEF:+91
+INSERT INTO `item_mods` VALUES (25583,2,36);      -- HP+36
+INSERT INTO `item_mods` VALUES (25583,5,29);      -- MP+29
+INSERT INTO `item_mods` VALUES (25583,8,10);      -- STR+10
+INSERT INTO `item_mods` VALUES (25583,9,10);      -- DEX+10
+INSERT INTO `item_mods` VALUES (25583,10,12);     -- VIT+12
+INSERT INTO `item_mods` VALUES (25583,11,1);      -- AGI+1
+INSERT INTO `item_mods` VALUES (25583,12,29);     -- INT+29
+INSERT INTO `item_mods` VALUES (25583,13,23);     -- MND+23
+INSERT INTO `item_mods` VALUES (25583,14,21);     -- CHR+21
+INSERT INTO `item_mods` VALUES (25583,29,6);      -- Magic Defense+6
+INSERT INTO `item_mods` VALUES (25583,30,26);     -- Magic Accuracy+26
+INSERT INTO `item_mods` VALUES (25583,31,53);     -- Magic Evasion+53
+INSERT INTO `item_mods` VALUES (25583,68,33);     -- Evasion+33
+INSERT INTO `item_mods` VALUES (25583,311,52);    -- Magic Damage+52
+INSERT INTO `item_mods` VALUES (25583,384,40);    -- Haste+40
+INSERT INTO `item_mods` VALUES (25584,1,96);      -- Mallquis Chapeau +1 DEF:+96
+INSERT INTO `item_mods` VALUES (25584,2,36);      -- HP+36
+INSERT INTO `item_mods` VALUES (25584,5,29);      -- MP+29
+INSERT INTO `item_mods` VALUES (25584,8,10);      -- STR+10
+INSERT INTO `item_mods` VALUES (25584,9,10);      -- DEX+10
+INSERT INTO `item_mods` VALUES (25584,10,19);     -- VIT+19
+INSERT INTO `item_mods` VALUES (25584,11,1);      -- AGI+1
+INSERT INTO `item_mods` VALUES (25584,12,36);     -- INT+36
+INSERT INTO `item_mods` VALUES (25584,13,23);     -- MND+23
+INSERT INTO `item_mods` VALUES (25584,14,21);     -- CHR+21
+INSERT INTO `item_mods` VALUES (25584,28,12);     -- Magic Attack+12
+INSERT INTO `item_mods` VALUES (25584,29,6);      -- Magic Defense+6
+INSERT INTO `item_mods` VALUES (25584,30,38);     -- Magic Accuracy+38
+INSERT INTO `item_mods` VALUES (25584,31,53);     -- Magic Evasion+53
+INSERT INTO `item_mods` VALUES (25584,68,33);     -- Evasion+33
+INSERT INTO `item_mods` VALUES (25584,311,52);    -- Magic Damage+52
+INSERT INTO `item_mods` VALUES (25584,384,40);    -- Haste+40
```

Comment:
For future reference: `haste +4%` saves me from having to verify you used the mod correctly to get 40/1024ths haste to get 4% and weren't trying to apply something crazy like 40% haste.